### PR TITLE
Connect Angor to UniSat Wallet

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Angor/UniSatWallet/Exceptions/NoUniSatWalletException.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UniSatWallet.Exceptions
```
Comment (DavidGershony) [inline src/Angor/UniSatWallet/Exceptions/NoUniSatWalletException.cs:7]: If we have a project for each wallet that could become a very big solution, perhaps a project for wallet connectors with a connector and models file per wallet?

**Review by DavidGershony — COMMENTED** [review]
We need to discuss the project per wallet thing..

**Comment by dangershony** [comment]
I wonder if we should hold this PR till we get the platform to a stable point, though it is all nicely isolated to its own project

**Comment by dangershony** [comment]
Closing for now as this is lower priority

**Pull request closed without merging**